src/main.js

```diff
@@ -78,12 +77,59 @@ function switchScreens(closingWindow, openingWindow){
 
 function userInput() {
   currentPoster = new Poster(userPosterImage.value, userPosterTitle.value, userPosterQuote.value);
-  posterImage.src = currentPoster.imageURL;
-  posterTitle.innerHTML = currentPoster.title;
-  posterQuote.innerHTML = currentPoster.quote;
-  images.push(currentPoster.imageURL);
-  titles.push(currentPoster.title);
-  quotes.push(currentPoster.quote);
+  var error = errorMessages();
+  if (!error) {
+    posterImage.src = currentPoster.imageURL;
+    posterTitle.innerHTML = currentPoster.title;
+    posterQuote.innerHTML = currentPoster.quote;
+    images.push(currentPoster.imageURL);
+    titles.push(currentPoster.title);
+    quotes.push(currentPoster.quote);
+    switchScreens('.main-poster', '.poster-form');
+    clearForm();
+    removeAllErrors();
+  }
+}
+
+function errorMessages() {
```

Comment:
The naming here is fine for now, but, in the future, try to do something a little more descriptive. A couple of options could be:

```javascript
function checkForErrorMessages() { ... };

function hasErrorMessages() { ... };

function isFormValid() { ... };

function validateForm() { ... };
```

This helps show that the return value of the function we should expect is a boolean. Using the `is/has/was` naming convention is pretty standard and really helpful for other devs when skimming through the code.